cli/src/commands/branch.rs

```diff
@@ -12,13 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::collections::HashSet;
+use std::collections::{HashMap, HashSet};
 use std::fmt;
 use std::io::Write as _;
 
 use clap::builder::NonEmptyStringValueParser;
 use itertools::Itertools;
-use jj_lib::git;
+use jj_lib::git::{self};
```

Comment:
nit: omit `::{self}`

(I'm surprised that `rustfmt` doesn't complain about that.)